Bulk content: Activation checklist content (#4137)
PostHog/posthog.com: contents/blog/five-essential-posthog-apps.md

---

> date: 2022-09-26

Reviewer: ```suggestion
date: 2022-10-04
```

> date: 2022-09-10

Author: ```suggestion
date: 2022-10-10
```

> title: "Getting started: Your five essential PostHog apps"

Author: Tricky though, as these aren't the most popular.

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ["joe-martin"]
featuredImage: ../images/blog/posthog-array-blog.png
featuredImageType: full
categories: ["Guides"]
---

> PostHog apps are a powerful, but ill-defined part of the platform. They’re powerful because they can do almost anything — and they’re ill-defined because they do almost anything.

Author: ```suggestion
PostHog apps are a powerful, but hard to explain part of the platform. They’re powerful because they can do almost anything — and they’re hard to explain because they do almost anything.
```
